lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
I really don't like this change.
It obscures that these are all class methods.
I'd like to see the `self.` prefixes back.